mne_bids_pipeline/tests/test_documented.py

```diff
@@ -26,7 +26,7 @@ def test_options_documented():
     settings_path = root_path.parent / "docs" / "source" / "settings"
     assert settings_path.is_dir()
     in_doc = set()
-    key = "::: mne_bids_pipeline.config."
+    key = "::: mne_bids_pipeline._config."
```

Comment:
This probably means that all `docs/` options need to be updated to reflect this, and all `changelog` entries...

Maybe it's worth it to make this private?

Reply:
Oh you've done this already above. I guess it's okay then

Reply:
I didn't update any changelog entries though 🤔

Reply:
Yes one problem with `mkdocs` is that there is no way AFAIK to treat warnings as errors, so I only notice bad links when I build locally and see the warnings

Reply:
I jist pushed those changes